step-framework-parent/step-framework-timeseries/src/main/java/step/core/timeseries/metric/MetricType.java

```diff
@@ -0,0 +1,80 @@
+package step.core.timeseries.metric;
+
+import step.core.accessors.AbstractIdentifiableObject;
+
+import java.util.List;
+import java.util.Map;
+
+public class MetricType extends AbstractIdentifiableObject {
+    
+    private String name;
+    private String label;
+    private String unit;
+    private List<MetricAttribute> attributes;
+    private String groupingAttribute;
```

Comment:
@jeromecomte groupingAttribute is the default grouping. The user can then switch to any attribute using the attributes list.

Reply:
In this case we should rename it defaultGroupingAttributes. As we might group by multiple attributes per default I assume that this field should be a Set. no?

Reply:
yes. in your specification all charts seemed to be grouped by one dimension only. If we want to allow multiple of them, yes. In this case though, the 'grouping' menu I created does not make sense anymore - because there should be a way to select more (I will find a way for this).